src/elements/featured-videos.html

```diff
@@ -113,44 +113,48 @@
     </style>
 
     <div class="container">
-      <div class="videos-header">
-        <h4 class="heading">{$ featuredVideosTitle $}</h4>
-        <div class="controls">
-          <paper-icon-button icon="icons:chevron-left" on-tap="shiftContentLeft"></paper-icon-button>
-          <paper-icon-button icon="icons:chevron-right" on-tap="shiftContentRight"></paper-icon-button>
+      <template is="dom-if" if="[[_videos]]">
```

Comment:
It's better to use `hidden` attribute on `<featured-videos/>` element in `home-page.html`. 
https://github.com/gdg-x/hoverboard/blob/master/src/pages/home-page.html#L25